### ISSUE TITLE
Delete button is visible when min parameter is set and collection have min elements

### ISSUE THREAD

**Issue by Darknessshade** [ninsuo/symfony-collection#63]
Hello
Remove the Delete button when there are min + 1 elements before doDelete.

```javascript
#Line 524 after toDelete.remove();
if( elements.length == settings.min + 1 ){
    collection.find('.' + settings.prefix + '-remove').remove();
}
```

**Comment by ninsuo** [comment]
Thank you, this was fixed in 2c595a536fa58fcfc438389ffcd97c8651044820 (side effects of fade_out option by default set to true)

**Comment by ninsuo** [comment]
http://symfony-collection.fuz.org/symfony3/options/numberCollectionElements
